static/app/views/starfish/components/tableCells/spanDescriptionCell.tsx

```diff
@@ -131,30 +131,38 @@ function QueryDescriptionOverlay({
 
   return description ? (
     <PositionWrapper zIndex={theme.zIndex.tooltip} {...hoverOverlayProps.overlayProps}>
```

Comment:
Howcome we didn't just use HoverCard for this btw?

Reply:
probably some complication on how loading works huh

Reply:
What's `HoverCard`!?